### PR TITLE
UF-8534: Update API to handle internal/external

### PR DESCRIPTION
**Pull request by cheezi0747**
Also includes fixes to warnings, errors and formatting

### CODE REVIEW

File: src/main/java/se/sundsvall/webmessagecollector/api/model/MessageDTO.java [hunk 1]
```diff
@@ -43,16 +43,19 @@ public class MessageDTO {
 	@Schema(description = "Firstname of the poster ", example = "Test")
 	private final String firstName;
 
-	@Schema(description = "If the message was utbound from a manager or inbound from a customer ", example = "Testorsson")
+	@Schema(description = "If the message was outbound from a manager or inbound from a customer ", example = "Testorsson")
```
Comment (AndreSoderlund) [inline src/main/java/se/sundsvall/webmessagecollector/api/model/MessageDTO.java:46]: Something feels off about this description, same on row 52.

File: src/test/java/se/sundsvall/webmessagecollector/configuration/ShedlockUsageTest.java [hunk 2]
```diff
@@ -70,10 +71,11 @@ private Map<String, List<Method>> getMethodsAnnotatedWith(final BeanDefinition c
 				// move to the upper class in the hierarchy in search for more methods
 				klazz = klazz.getSuperclass();
 			}
-			return Map.of(candidate.getBeanClassName(), methods);
-		} catch (ClassNotFoundException e) {
+			return Map.of(Objects.requireNonNull(candidate.getBeanClassName()), methods);
```
Comment (AndreSoderlund) [inline src/test/java/se/sundsvall/webmessagecollector/configuration/ShedlockUsageTest.java:74]: Is the "Objects.requireNonNull" necessary? 

The we pass an object that is null and NPE is thrown either way, with or without it.

File: src/main/java/se/sundsvall/webmessagecollector/service/MessageService.java [hunk 3]
```diff
@@ -35,8 +32,8 @@ public MessageService(final MessageRepository repository, final MessageAttachmen
 		this.attachmentRepository = attachmentRepository;
 	}
 
-	public List<MessageDTO> getMessages(final String familyId) {
-		return toMessageDTOs(repository.findAllByFamilyId(familyId));
+	public List<MessageDTO> getMessages(final String familyId, final String instance) {
+		return toMessageDTOs(repository.findAllByFamilyIdAndInstance(familyId, Instance.fromString(instance)));
 	}
```
Comment (AndreSoderlund) [inline src/main/java/se/sundsvall/webmessagecollector/service/MessageService.java:37]: We should probably validate the instance value. With this current setup i'm guessing an invalid string will always return an empty list. Which might be OK?

File: src/test/java/se/sundsvall/webmessagecollector/configuration/ShedlockUsageTest.java (hunk 2 above)
Comment (cheezi0747) [reply src/test/java/se/sundsvall/webmessagecollector/configuration/ShedlockUsageTest.java:74]: Yes.

File: src/main/java/se/sundsvall/webmessagecollector/service/MessageService.java (hunk 3 above)
Comment (AndreSoderlund) [reply src/main/java/se/sundsvall/webmessagecollector/service/MessageService.java:37]: I don't know if it is OK. Just a heads up.

**Pull request merged**